README.md

```diff
@@ -77,6 +77,43 @@ This is where you work on the code-base that will be used by yours and other Hyr
     <p>By moving to Docker, we are encoding the documentation steps for standing up a Hyrax-engine development environment.</p>
 </aside>
 
+### Installing Analytics
+
+Hyrax supports your choice of either Google Analytics or Matomo.  To enable analytics tracking and reporting features, follow the directions below.
+
+Enable Analytics Features
+
+In your .env file, set HYRAX_ANALYTICS to true, set either 'google' or 'matomo' for  HYRAX_ANALYTICS_PROVIDER, and set the date you would like reporting to start (ANALYTICS_START_DATE).
```

Comment:
use of `.env` isn't prescribed by Hyrax; reference to it may confuse folks deploying into environments where it's not used. it's probably better to say something like "analytics can be configured using environment variables..." instead.